{"name":"userReg","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"C:\\Users\\dme595\\eclipse-workspace\\ECOM_WIZARD\\src\\excelExportAndFileIO\\ExportExcel.xlsx (The process cannot access the file because it is being used by another process)","trace":"java.io.FileNotFoundException: C:\\Users\\dme595\\eclipse-workspace\\ECOM_WIZARD\\src\\excelExportAndFileIO\\ExportExcel.xlsx (The process cannot access the file because it is being used by another process)\r\n\tat java.base/java.io.FileOutputStream.open0(Native Method)\r\n\tat java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)\r\n\tat java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)\r\n\tat java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)\r\n\tat pages.userRegistration.tempMailVerification(userRegistration.java:104)\r\n\tat testCases.loginTest.userReg(loginTest.java:55)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"TestCase : User Registration","steps":[],"attachments":[{"name":"saveFailureScreenShot","source":"f6602041-1ebb-4542-8f90-43a208a095b0-attachment"},{"name":"userReg failed and screenshot taken!","source":"502babc0-8f7f-449b-a177-4fd236ec481b-attachment","type":"text/plain"}],"parameters":[],"start":1679046406955,"stop":1679046413705,"uuid":"65772bdb-527a-4aa6-90d0-a88611d63d85","historyId":"367f6f98774eec99d532635782b4b0bf","fullName":"testCases.loginTest.userReg","labels":[{"name":"package","value":"testCases.loginTest"},{"name":"testClass","value":"testCases.loginTest"},{"name":"testMethod","value":"userReg"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testCases.loginTest"},{"name":"host","value":"SEZ-WS-47"},{"name":"thread","value":"20124@SEZ-WS-47.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}